README.md

```diff
@@ -139,6 +140,27 @@ EXAMPLE
 
 _See code: [build/cli/src/commands/checksum.js](https://github.com/airtasker/spot/blob/v0.2.18/build/cli/src/commands/checksum.js)_
 
+## `spot docs SPOT_CONTRACT`
+
+Preview Spot contract documentation
```

Comment:
can this document be extended? I feel like it needs to explain which url to visit to access the docs.

Reply:
The command will output:
```
Documentation server started on port 8080
Open http://localhost:8080 to view documentation
```
Is this sufficient? or do you think the Readme also needs more information?

Reply:
is it this or `http://localhost:8080/contract`?

Reply:
Would be good to put it into readme